pages/search.js

```diff
@@ -85,19 +84,23 @@ export default function Search({ data: { users, tags, randUsers } }) {
       const nameLower = user.name.toLowerCase();
       const usernameLower = user.username.toLowerCase();
       const userTagsString = user.tags.join(", ").toLowerCase();
-      
+      const userLocationString = [user.location.provided]
```

Comment:
This should be `const userLocationString = user.location ? user.location.name.toLowerCase() : "";`

Reply:
Fixed